feat: websockets support (#172)
input-output-hk/atala-prism-mediator: README.md

- `KEY_AGREEMENT_X` - is the key agreement public key (MUST be a X25519 OKP key type).
- `KEY_AUTHENTICATION_D` - is the key authentication private key (MUST be an Ed25519 OKP key type).
- `KEY_AUTHENTICATION_X` - is the key authentication public key (MUST be an Ed25519 OKP key type).

> - `SERVICE_ENDPOINTS` - is the list of endpoints of the mediator split by ';'. Where the mediator will be listening to incoming DID Comm messages.

Reviewer: ```suggestion
- `SERVICE_ENDPOINTS` - is the list of endpoints of the mediator split by ';' where the mediator will listen to incoming DIDComm messages.
```
